### DISCUSSION TITLE
OPENCLASH 一直转圈自动刷新是打开的也无法关闭

### DISCUSSION THREAD

**Discussion by kevinyang68** [vernesong/OpenClash#2469] [General]
不知道是不是因为自动刷新是开的，我点了关闭也无用，切回运行状态就变成自动刷新了，该页面始终在转圈。显示等待OPENWRT 的网关响应。寻求帮助。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。。

**Comment by LittleNewton** [comment]
我是OpenWrt 22.03 snapshot版本，打开OpenClash页面，网页图标也是一直转圈，但是不影响使用。不知道后台是不是在加载什么。
